django_ical/views.py

```diff
@@ -82,7 +82,7 @@ def __call__(self, request, *args, **kwargs):
         except ObjectDoesNotExist:
             raise Http404('Feed object does not exist.')
         feedgen = self.get_feed(obj, request)
-        response = HttpResponse(content_type=feedgen.mime_type)
+        response = HttpResponse(content_type='text/calendar, text/x-vcalendar, application/hbs-vcs')
```

Comment:
What is the current mimetype? Should this ideally be implemented in `feedgen.mime_type` or appended to it?

If any changes are made into the feedgen, then the type is not updated here, but requires manual patching, so I'm wondering if this should be something like:

    calendar_mime_type = 'text/calendar, text/x-vcalendar, application/hbs-vcs'
    content_type = ', '.join([calendar_mime_type, feedgen.mime_type])
    response = HttpResponse(content_type=content_type)